packages/admin/cms-admin/src/dependencies/DependencyList.tsx

```diff
@@ -104,7 +104,8 @@ export const DependencyList = ({ query, variables }: DependencyListProps) => {
                         apolloClient,
                         id: row.id,
                     });
-                    return contentScope.match.url + path;
+                    const hasScopeInBasePath = !!matchPath(path, contentScope.match);
+                    return hasScopeInBasePath ? path : contentScope.match.url + path;
```

Comment:
`resolvePath` can now either return the URL including the scope path or without the scope path. 

This is theoretically breaking but in practice this is the only place it's used